core/src/i18n/en.yaml

```
@@ -25,6 +31,8 @@ api-rules:
   messages:
     create-failed: Failed to create the API Rule
     created: API Rule created
+    description: Expose Services outside the cluster with API Rules.
+    not-matching-results: Couldn't find API Rules matching this query.
     updated: API Rule updated
   services:
     error-load: Couldn't load service list
```

Is this the "Kubernetes concept" kind of Service (capitalized) or the "Service Management" kind of service?
If Kubernetes Service, we need: 

```suggestion
    error-load: Couldn't load Service list
```